### PR TITLE
Django app as a container

### PR DESCRIPTION
**Pull request by DevAseel**
- Containerized Django app.

### CODE REVIEW

File: components/api/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+RUN pip install psycopg2-binary --no-binary psycopg2-binary
+
+# installing packages
+WORKDIR /usr/src/app
+COPY ./requirements.txt .
+RUN [ "python3", "-m", "pip", "install", "-r", "requirements.txt" ] 
+
+
+# copying files & folders
+COPY ./ ./
```
Comment (specialistvlad) [inline components/api/Dockerfile:21]: @DevAseel please avoid this pattern and copy at least some folders and files required.

Comment (DevAseel) [reply components/api/Dockerfile:21]: @specialistvlad all of the files are part of django configuration. I've followed the [official docker documentation for Django](https://github.com/docker/awesome-compose/tree/master/official-documentation-samples/django/).

Comment (specialistvlad) [reply components/api/Dockerfile:21]: Ok, that's fine then

**Comment by specialistvlad** [comment]
Tested
Screenshot 2023-03-15 at 12 08 14 PM

**Pull request merged**